.expeditor/scripts/build-infra-deb.sh

```diff
@@ -0,0 +1,187 @@
+#!/bin/bash
+
+set -euo pipefail
+
+validate_env_vars() {
+    if [ -z "${CHEF_INFRA_MIGRATE_TAR_DEB:-}" ] || [ -z "${CHEF_INFRA_HAB_TAR_DEB:-}" ]; then
```

Comment:
Remove `_DEB` from the environment variable to align with the other build script for RPM.